requirements.txt

```
@@ -5,7 +5,7 @@ cookiecutter~=1.7.0
 dynaconf>=3.1.2,<4.0.0
 fsspec>=2021.4, <=2022.1
 gitpython~=3.0
-importlib_metadata>=4.4 # Compatible with Python 3.10 importlib.metadata
+importlib_metadata>=3.6; python_version < '3.10' # The "selectable" entry points were introduced in `importlib_metadata` 3.6 and Python 3.10.
```

```suggestion
importlib_metadata>=3.6  # The "selectable" entry points were introduced in `importlib_metadata` 3.6 and Python 3.10.
```
The alternative is to do something like:

```python
try:
    import importlib.metadata as importlib_metadata
except ImportError:
    import importlib_metadata
```

(The `importlib_metadata` or some other name needs to be kept consistent, so that the patching in tests could work.)

This would avoid the "unnecessary" `importlib_metadata` import for Python 3.10, but it seems like this would keep complexity lower, and `importlib_metadata` shouldn't drag in extra requirements (to affect compatibility) anyway.